api/strategies/openidStrategy.js

```diff
@@ -194,6 +236,30 @@ async function setupOpenId() {
             }
           }
 
+          let customOpenIdData = new Map();
+          if (process.env.OPENID_CUSTOM_DATA) {
+            const dataMapper = OpenIdDataMapper.getMapper(process.env.OPENID_PROVIDER.toLowerCase());
+            customOpenIdData = await dataMapper.mapCustomData(tokenset.access_token, process.env.OPENID_CUSTOM_DATA);
+            const tokenBasedRoles =
+                  requiredRole &&
+                  extractRolesFromToken(
+                    safeDecode(requiredRoleTokenKind === 'access' ? tokenset.access_token : tokenset.id_token),
+                    requiredRoleParameterPath,
+                  );
+            if (tokenBasedRoles && tokenBasedRoles.length) {
+              customOpenIdData.set('roles', tokenBasedRoles);
+            } else {
+              logger.warn('[openidStrategy] tokenBasedRoles is missing or invalid.');
+            }
+          }
+
+          const token = requiredRoleTokenKind === 'access' ? tokenset.access_token : tokenset.id_token;
+          const decodedToken = safeDecode(token);
+          const tokenBasedRoles = extractRolesFromToken(decodedToken, requiredRoleParameterPath);
+          const isAdmin = tokenBasedRoles.includes(adminRole);
```

Comment:
Can we have multiple adminRoles that at the end will make the user admin? Or it is not a good idea ?

Reply:
@geodanchev I'm sorry that this is not on my highest priority list. after I am finished with updating the docs for translation I will finish this.

Reply:
off course you can have multiple admin roles.

Reply:
Ok, so my assumption is that `const isAdmin = tokenBasedRoles.includes(adminRole);` should be changed to `const isAdmin = adminRoles.some(role => tokenBasedRoles.includes(role));`
What do you think ?